### PR TITLE
update adrien joly

### PR DESCRIPTION
**Pull request by adrienjoly**

### CODE REVIEW

File: baggers.js [hunk 1]
```diff
@@ -8459,26 +8459,11 @@ var data = {
...
-            'Lean Startup',
-            'Bootstrapping',
-            'CEO',
-            'CTO',
-            'Trello',
-          ],
-          lang: ['fr'],
-        },
-      ],
+      sessions: [],
```
Comment (dadoonet) [inline baggers.js]: But if you don't have sessions anymore you should better remove your profile no?

**Review by dadoonet — CHANGES_REQUESTED** [review]

File: baggers.js (hunk 1 above)
Comment (adrienjoly) [reply baggers.js]: That's a fair point! => I just added 3 sessions. What do you think?

**Review by dadoonet — APPROVED** [review]
LGTM.

**Comment by adrienjoly** [comment]
Merci David! ^^

**Pull request merged**